tests/plugin_tests/test_feeds.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from feedparser import FeedParserDict
+from mock import patch
+
+
+@pytest.fixture()
+def mock_feedparse():
+    with patch('feedparser.parse') as mock:
```

Comment:
Can this not be done with a single with? ie, `with patch('feedparser.parse') as mock, patch('cloudbot.util.web.try_shorten', lambda x: x):`? Saves an indentation level. Though may increase line length

Reply:
tbh this should be a second fixture entirely